core/unit_test/TestViewIsAssignable.hpp

```diff
@@ -92,8 +92,18 @@ TEST(TEST_CATEGORY, view_is_assignable) {
                           View<double*, left, d_exec>>::test(false, false, 10);
 
   // Layout assignment
+  Impl::TestAssignability<View<int, left, d_exec>,
+                          View<int, right, d_exec>>::test(true, true);
```

Comment:
No right -> left?

Reply:
I guess we could add them everywhere, but mostly this is about mismatch, not any specific layouts.